charts/dex/values.yaml

```diff
@@ -120,6 +120,9 @@ ingress:
   #  - secretName: chart-example-tls
   #    hosts:
   #      - chart-example.local
+# Sets Ingress/ingressClassName. This way ingress resources are able to bound specific ingress-controllers. Kubernetes version >=1.18 required.
```

Comment:
```suggestion
  # Sets Ingress/ingressClassName. This way ingress resources are able to bound specific ingress-controllers. Kubernetes version >=1.18 required.
```

Reply:
Actually, it also needs the `# -- ` part to show up in the generated documentation. Please also add a new line before this.